app/src/main/java/com/pajato/android/gamechat/fragment/TTTFragment.java

```diff
@@ -3,47 +3,84 @@
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
 import android.support.v4.content.ContextCompat;
+import android.util.Log;
 import android.util.TypedValue;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.Button;
 import android.widget.TextView;
 
+import com.firebase.client.DataSnapshot;
+import com.firebase.client.Firebase;
+import com.firebase.client.FirebaseError;
+import com.firebase.client.GenericTypeIndicator;
+import com.firebase.client.ValueEventListener;
 import com.pajato.android.gamechat.R;
 import com.pajato.android.gamechat.game.GameManager;
 
+import java.util.HashMap;
 import java.util.Scanner;
 
 public class TTTFragment extends Fragment {
```

Comment:
A JavaDoc comment should be added.  Every class, enum, interface should have a JavaDoc comment even if it is a one liner: /** A cool class to do amazing stuff! */